tests/by-util/test_rm.rs

```diff
@@ -263,3 +263,33 @@ fn test_rm_no_operand() {
     ucmd.fails()
         .stderr_is("rm: error: missing an argument\nrm: error: for help, try 'rm --help'\n");
 }
+
+#[test]
+fn test_rm_verbose_slash() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    let dir = "test_rm_verbose_slash_directory";
+    let file_a = &format!("{}/test_rm_verbose_slash_file_a", dir);
+
+    at.mkdir(dir);
```

Comment:
please run rustfmt

Reply:
I thought I did run it :o maybe the change got lost bisecting unrelated formatting, will run it again :)

Reply:
Figured it out: `cargo fmt` does not run the formatter on tests so I had to manually run `rustfmt` on the specific file

https://github.com/rust-lang/rustfmt/issues/803

Should be fixed now